src/app/containers/ArticleMain/__snapshots__/index.test.jsx.snap

```diff
@@ -3,6 +3,7 @@
 exports[`ArticleMain should render a news article correctly 1`] = `
 <RequestContextProvider
   bbcOrigin="https://www.test.bbc.co.uk"
+  currentPath={null}
```

Comment:
Should this really be `null`? I'd expect `currentPath` to always have a value...